commands.go

```diff
@@ -106,11 +130,23 @@ func ctfCommandCallback(s *discordgo.Session, i *discordgo.InteractionCreate) {
 		}
 		respComponents = append(respComponents, actionRow)
 
+		leaveActionRow := discordgo.ActionsRow{
+			Components: []discordgo.MessageComponent{
+				discordgo.Button{
+					Label:    "Leave " + ctfName,
+					Style:    discordgo.SuccessButton,
+					Disabled: false,
+					CustomID: "ctf_leave",
+				},
+			},
+		}
+		leaveRespComponent = append(leaveRespComponent, leaveActionRow)
+
 		// Create a channel for the CTF that is locked to the created role
 		targetGuild, _ := s.Guild(i.GuildID)
 		everyoneID := targetGuild.Roles[0].ID // Per my assumption, @everyone is the first role - Jack
 
-		s.GuildChannelCreateComplex(i.GuildID, discordgo.GuildChannelCreateData{
+		channel, _ := s.GuildChannelCreateComplex(i.GuildID, discordgo.GuildChannelCreateData{
```

Comment:
I would add error handling here if possible, assuming that the second value is an error. If an error is detected, lets log it with `log.Println` and maybe return a message to the user in channel (ephemeral?)

Reply:
Added